### PR TITLE
Create temporary free tag

### LINKED ISSUE #882: Rework command `!free` for the new help system.
## Issue
With the newly introduced help systems, the old command `!free` is no longer relevant.

However, there are will be users who get confused and may post questions in the in-use questions / ask how to and where to get help.

## Proposal
We can revamp the `!free` command to:
- Output a short summary of how to claim a free channel.
- [Optional] Mention the current free channels they can claim.
- [Optional] Show a screenshot of the free channels and what they look like.

### PR DESCRIPTION
**Pull request by Akarys42**
Closes #882

This PR add a temporary `free` tag that indicate how the new help system works.

### CODE REVIEW

File: bot/resources/tags/free.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+We recently moved to a new help channel system. There are always 2 available help channels waiting to be claimed in the **Python Help: Available category**. In order to claim one, simply start typing your question into one of these channels. Once your question has been posted, you have claimed this channel, and the channel will be moved down to the **Python Help: Occupied category**.
```
Comment (ikuyarihS) [inline bot/resources/tags/free.md:1]:
```suggestion
**How to claim a channel**

We recently moved to a new help channel system. There are always 2 available help channels waiting to be claimed in the **Python Help: Available category**. In order to claim one, simply start typing your question into one of these channels. Once your question has been posted, you have claimed this channel, and the channel will be moved down to the **Python Help: Occupied category**.
```

File: bot/resources/tags/free.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
```
Comment (ikuyarihS) [inline bot/resources/tags/free.md:2]: I suggest changing the name to discord channel mention via id, that way if we ever decide to rename the channels, the content of the tag will be updated accordingly.
```suggestion
We recently moved to a new help channel system. There are always 2 available help channels waiting to be claimed in the **<#691405807388196926>**. In order to claim one, simply start typing your question into one of these channels. Once your question has been posted, you have claimed this channel, and the channel will be moved down to the **<#696958401460043776>**.
```

**Comment by lemonsaurus** [comment]
I think @kwzrd is making a lot of sense. I'd like to see us make those requested changes.

**Comment by kwzrd** [comment]
As there are still members trying to use the `!free` cmd and receiving no response, we'd like to have this merged sometime soon.

@Akarys42, do you think you'll be able to address the requested changes? If not, then that's ok, but please let us know. Thanks!

**Comment by Akarys42** [comment]
I'll try to find some time to rephrase it this afternoon

**Pull request closed without merging**